src/strategies/decentraland-wearable-rarity/README.md

```diff
@@ -0,0 +1,47 @@
+# Decentraland Wearable Rarity
+
+This strategy allows users to calculate the total amount of Decentraland wearables and apply a multiply to each rarity. Additionally you can limit the collections that will be take into account using a list of ids.
+
+## Example
+
+The should should look like this:
+
+```json
+{
+    "symbol": "WEARABLE",
+    "multipliers": {
+        "unique": 1000000,
+        "mythic": 100000,
+        "legendary": 10000,
+        "epic": 1000,
+        "rare": 100,
+        "uncommon": 10,
+        "common": 1
+    }
+}
+```
+
+If you want to restrict which collection is take into account should looks like this:
```

Comment:
Typos and change in grammar? "If you want to restrict which collection is taken into account, parameters should looks like this:"